src/dealDashboard/dealDiscussion/discussionThread/singleComment/singleComment.html

```diff
@@ -95,5 +95,9 @@
       </div>
       <pre>${comment.text}</pre>
     </section>
+    <div if.bind="loading['isDeleting ' + comment._id]" class="deleteInfo">
+      <i class="spinner fa fas fa-circle-notch fa-spin"></i>
```

Comment:
I think "spinner" and "fa-spin" may be redundant

Reply:
`.spinner` is for styling the color. `.fa-spin` is `Font-Awesome`'s built in for spinning animation.